### ISSUE TITLE
[SIGGRAPH_2022] Change of PAE model specification

### ISSUE THREAD

**Issue by NeuTTH** [sebastianstarke/AI4Animation#112]
Hi! great work. 

I am reading both the paper and code base concurrently and realize that encoder and decoder layer specifications are changed. In the paper [3.3 network training] it is mentioned to use both bath normalization and Tanh but in the most recent update it was changed to a [custom layer normalization](https://github.com/sebastianstarke/AI4Animation/blob/master/AI4Animation/SIGGRAPH_2022/PyTorch/Library/Utility.py) and [ELU](https://github.com/sebastianstarke/AI4Animation/blob/master/AI4Animation/SIGGRAPH_2022/PyTorch/PAE/PAE.py)? Is there any reason towards this change and what are the qualitative differences?

Thank you very much!
